### PR TITLE
Fix renumbering non visual properties

### PR DESCRIPTION
**Pull request by buchi**
- Add renumbering of non-visual image properties (pic:cNvPr)
- Also renumber non-visual drawing properties (wp:docPr) in headers and
  footers

Fixes: https://4teamwork.atlassian.net/browse/GEVER-508

### CODE REVIEW

**Comment by deiferni** [comment]
@buchi i temporarily changed the merge base to `embedded-images-with-external-ref` for my review

File: docxcompose/composer.py [hunk 1]
```diff
@@ -556,13 +557,43 @@ def renumber_bookmarks(self):
...
+            for doc_pr in doc_prs:
+                doc_pr.id = doc_pr_id
+                doc_pr_id += 1
+
+    def renumber_nvpicpr_ids(self):
+        # Ensure that non-visual image properties have a unique id
+        c_nv_prs = xpath(
+            self.doc.element.body, './/pic:cNvPr')
+        c_nv_pr_id = 1
+        for c_nv_pr in c_nv_prs:
```
Comment (deiferni) [inline docxcompose/composer.py:583]: you could use `enumerate(c_nv_prs, 1)` to make it a bit more "pythonic"

File: docxcompose/composer.py [hunk 2]
```diff
@@ -556,13 +557,43 @@ def renumber_bookmarks(self):
...
+            c_nv_pr.id = c_nv_pr_id
+            c_nv_pr_id += 1
+
+        parts = [
+            rel.target_part for rel in self.doc.part.rels.values()
+            if rel.reltype in [RT.HEADER, RT.FOOTER, ]
+        ]
+        for part in parts:
+            c_nv_prs = xpath(part.element, './/pic:cNvPr')
+            for c_nv_pr in c_nv_prs:
```
Comment (deiferni) [inline docxcompose/composer.py:593]: similar here

File: docxcompose/composer.py (hunk 1 above)
Comment (buchi) [reply docxcompose/composer.py:583]: Be aware that the counter continues when renumbering ids in headers and footers to have unique ids across parts.

**Pull request merged**